JS.md

```diff
@@ -205,6 +205,25 @@ to the Object `{ module: String(i.module_name), name: String(i.item_name), kind:
 
 Note: other fields like `signature` may be added in the future.
 
+### `WebAssembly.Module.userSections`
```

Comment:
We moved to calling these "user" sections in wabt, and in my proposed llvm binutils patch.

I think "user" is fine, and has more meaning than either "named" or "unkown" which don't seem to convey much to me.   We're all used to "user_data" pointers in callbacks.